### ISSUE TITLE
Merge project with bolt-shorturl

### ISSUE THREAD

**Issue by madc** [LulububuSoftwareGmbH/boltredirector#18]
Just a thought: I wrote a plugin that has a similar functionality some time ago.
[bolt-shorturl](https://github.com/AnimalDesign/bolt-shorturl) provides a custom fieldtype that allows to setup a shorturl per entry. From the docs:
"This plugin provides a new 'shorturl' fieldtype for contenttypes and redirects accordingly. When set up, it redirects i.e. from http://yourhost/s/short to http://yourhost/page/this-is-my-long-slug."

Maybe we could merge these plugins, as they are quite similar. Also, not sure, if the whole custom fieldtype thing is still necessary.

**Comment by mikerockett** [comment]
The thing is that you'd need to consider multiple source URIs mapping to the same entry/page/content. That particular package would have to be able to allow an entry to specify multiple sources, and I'd also recommend a separate view that brings them all together in a single list for easy-access purposes.

**Comment by SvanteRichter** [comment]
Hmm, I think that they serve two different audiences, the shorturl one is mainly meant for content authors, right? This extension is (IMO) mainly meant for developers trying to migrate a large number of old urls to a new bolt site.

I think it might be good if they shared a common base for handling redirects though, If my new work does not end up being very stressful I should be able to manage to put that out in the comming months.

**Comment by mikerockett** [comment]
I'd agree with that too. Sure, they could be interchangable, as suggested. So either share a common base, or perhaps incorporate the Shorturls package into this one...

**Comment by madc** [comment]
I totally agree, that they serve different audiences, but the overhead of combining them seems quite minimal to me. My suggestion would be to take the custom fieldtype from my extension and integrate it into this one. That would mean, as a developer I still can configure any redirection over my config file and as editor I can use the custom fieldtype.
